### PR TITLE
Grammar and test

### PR DESCRIPTION
**Pull request by henneboy**
The syntax is now (mostly) done

### CODE REVIEW

File: ParserTester/ParseOK/ArithExpr/ScaleVector.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿/* henrik wants to throw this out the window with force */
```
Comment (henneboy) [inline ParserTester/ParseOK/ArithExpr/ScaleVector.txt:1]: Maybe change this comment

**Review by henneboy — COMMENTED** [review]
Inappropriate comment should be removed :)

**Review by Casper-NS — APPROVED** [review]
lgtm

File: SableCC/Generated/GGcodeGrammar.sablecc [hunk 2]
```diff
@@ -10,47 +12,169 @@ Helpers
   letter  = lowercase | uppercase;
   idletter = letter | '_';
   idchar  = letter | '_' | digit;
+  xyz = 'x' | 'y' | 'z' | 'X' | 'Y' | 'Z';
```
Comment (Mati-AAU) [inline SableCC/Generated/GGcodeGrammar.sablecc:15]: We shouldnt allow both small and big characters for xyz

**Review by Mati-AAU — CHANGES_REQUESTED** [review]
do that one thing tyty

**Review by Mati-AAU — APPROVED** [review]
lgtm
ThumbsUpLikeGIF

File: SableCC/Generated/GGcodeGrammar.sablecc (hunk 2 above)
Comment (Casper-NS) [reply SableCC/Generated/GGcodeGrammar.sablecc]: It has been fixed

File: ParserTester/ParseOK/ArithExpr/ScaleVector.txt (hunk 1 above)
Comment (Casper-NS) [reply ParserTester/ParseOK/ArithExpr/ScaleVector.txt]: fixed

**Pull request merged**